apps/k8s-io/configmap-nginx.yaml

```
@@ -329,8 +329,8 @@ data:
         server_name sbom.k8s.io sbom.kubernetes.io
         listen 80;
 
-        rewrite ^/(.*)?/release$ https://storage.googleapis.com/kubernetes-release/release/$1/kubernetes-release.spdx redirect;
-        rewrite ^/(.*)?/source$  https://storage.googleapis.com/kubernetes-release/release/$1/kubernetes-source.spdx redirect;
+        rewrite ^/(.*)?/release(.*)?$ https://storage.googleapis.com/kubernetes-release/release/$1/kubernetes-release.spdx$2 redirect;
+        rewrite ^/(.*)?/source(.*)?$  https://storage.googleapis.com/kubernetes-release/release/$1/kubernetes-source.spdx$2 redirect;
```

Nit: given that we already know possible extensions (`.sig` and `.cert`), should we limit this regex to these two extensions? For example, something like:

```suggestion
        rewrite ^/(.*)?/release(\.cert|\.sig)?$ https://storage.googleapis.com/kubernetes-release/release/$1/kubernetes-release.spdx$2 redirect;
        rewrite ^/(.*)?/source(\.cert|\.sig)?$  https://storage.googleapis.com/kubernetes-release/release/$1/kubernetes-source.spdx$2 redirect;
```

One more benefit from this is that potentially malicious/spam requests are going to be terminated by nginx instead of getting routed to the origin bucket.

---

Changed as suggested, but added the SHA extensions as well.